File: Bridge/Resources/.generated/system/iO/iOException.js
    Bridge.define("System.IO.IOException", {
        inherits: [System.SystemException],
        ctors: {
            ctor: function () {
                this.$initialize();
                System.SystemException.ctor.call(this, "I/O Error");
            },
            $ctor1: function (message) {
                this.$initialize();
                System.SystemException.ctor.call(this, message);
            },
            $ctor2: function (message, innerException) {
                this.$initialize();
                System.SystemException.ctor.call(this, message, innerException);
            },
            $ctor3: function (message, hresult) {
                this.$initialize();
                System.SystemException.ctor.call(this, message);
            }
        }
    });

File: Bridge/System/IO/IOException.cs
using Bridge;

namespace System.IO
{
    public class IOException : SystemException
    {
        public IOException()
            : base("I/O Error")
        {
        }

        public IOException(string message)
            : base(message)
        {
        }

        public IOException(string message, Exception innerException)
            : base(message, innerException)
        {
        }
        
        public IOException(string message, int hresult)
            : base(message)
        {
        }
    }
}